requests/src/test/scala/weco/api/requests/RequestingScenarioTest.scala

```diff
@@ -1428,77 +1427,6 @@ class RequestingScenarioTest
     api.routes
   }
 
-  def createListHoldsRequest(patron: SierraPatronNumber): HttpRequest =
```

Comment:
All this code has been moved unmodified to SierraServiceFixture, so it can be used in RequestingScenarioTest.